doc/source/management/data_at_rest_encryption.rst

```diff
@@ -7,58 +7,207 @@ Data at Rest Encryption
+server must be started with the ``--bootstrap`` option.
+
+If the variable :variable:`innodb_sys_tablespace_encrypt` is set to ON and the
+server has been started in the bootstrap mode, you may create an encrypted table
+as follows:
+
+.. code-block:: guess
+
+   mysql> CREATE TABLE ... TABLESPACE=innodb_system ENCRYPTION='Y'
+
+.. note::
+
+   You cannot encrypt existing tables in the System tablespace.
+
+It is not possible to convert the system tablespace from encrypted to
+unencrypted or vice versa. A new instance should be created and user tables must
+be transferred to the desired instance.
+
+You can encrypt the already encrypted InnoDB system tablespace (key rotation)
+with a new master key by running the following ``ALTER INSTANCE`` statement:
+
+.. code-block:: guess
+
+   mysql> ALTER INSTANCE ROTATE INNODB MASTER KEY
+
+.. rubric:: Doublewrite Buffers
 
+The two types of doublewrite buffers used in |Percona Server| are encrypted
+differently.
+
+When the InnoDB system tablespace is encrypted, the ``doublewrite buffer`` pages
+are encrypted as well. The key which was used to encrypt the InnoDB system
+tablespace is also used to encrypt the doublewrite buffer.
+
+|Percona Server| encrypts the ``parallel doublewrite buffer`` with the respective
+tablespace key. Only encrypted tablespace pages are written as encrypted in the
+parallel doublewrite buffer. Unencrypted tablespace pages will be written as
+unencrypted.
+
+.. important::
+
+   A server instance bootstrapped with the encrypted InnoDB system tablespace
+   cannot be downgraded. It is not possible to parse encrypted InnoDB system
+   tablespace pages in a version of |Percona Server| lower than the version
+   where the InnoDB system tablespace has been encrypted.
+
+
+System variables
+--------------------------------------------------------------------------------
+
+.. variable:: innodb_sys_tablespace_encrypt
+
+   :version 5.7.23-24: Implemented
+   :cli: ``--innodb-sys-tablespace-encrypt``
+   :dyn: No
+   :scope: Global
+   :vartype: Boolean
+   :default: ``OFF``
+
+Enables the encryption of the InnoDB System tablespace. It is essential that the
+server is started with the ``--bootstrap`` option.
+
+.. seealso::
+
+   |MySQL| Documentation: ``--bootstrap`` option
+      https://dev.mysql.com/doc/refman/5.7/en/server-options.html#option_mysqld_bootstrap
+
+.. variable:: innodb_parallel_dblwr_encrypt
+
+   :version 5.7.23-24: Implemented
+   :cli: ``--innodb-sys-tablespace-encrypt``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Boolean
+   :default: ``OFF``
+
+Enables the encryption of the parallel doublewrite buffer. For encryption, uses
+the key of the tablespace where the parallel doublewrite buffer is used.
+
+
+.. _innodb_general_tablespace_encryption:
+
+InnoDB General Tablespace Encryption
+================================================================================
+
+.. TODO: add a reference to *existing tablespace support* or reword
```

Comment:
Please make sure that source code comments are not the sole place where TODOs are tracked - i.e. file JIRA too

Reply:
The TODO block removed